script/translations/util.py

```diff
@@ -20,3 +21,14 @@ def get_lokalise_token():
         )
 
     return token_file.read_text().strip()
+
+
+def get_current_branch():
+    """Get current branch."""
+    return (
+        subprocess.run(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5338343)